pymc3/stats/__init__.py

```diff
@@ -0,0 +1,31 @@
+#   Copyright 2020 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Alias for the `stats` submodule from ArviZ.
+
+Diagnostics and auxiliary statistical functions are delegated to the ArviZ library, a general
+purpose library for "exploratory analysis of Bayesian models."
+See https://arviz-devs.github.io/arviz/ for details.
+"""
+import sys
+
+import arviz as az
+
+for attr in dir(az.stats):
+    obj = getattr(az.stats, attr)
+    if not attr.startswith("__"):
```

Comment:
We might want to import just the entries from `__all__`. However then `pymc3.stats` is not a fully-functional alias to `arviz.stats`.
Opinions?

Reply:
I think that's fine, we only want to be a high-level interface for beginners, more advanced users can use `arviz` directly. We also don't clobber our namespace more than we have to.